src/types/resolveSignatures.ts

```diff
@@ -217,6 +217,17 @@ export function resolveSignatures(ctx: CompilerContext, Ast: FactoryAst) {
             }
         }
 
+        const lastField = t.fields.at(-1);
```

Comment:
Wait, let's actually ban it for any field, not just the last one

Reply:
If you write `Slice? as remaining` not in the last field, compiler will give an error already

Reply:
```
Error: test/main.tact:75:5: The "as remaining" field can only be the last field of the message
  74 | message Name {
> 75 |     bar: Slice? as remaining;
           ^~~~~~~~~~~~~~~~~~~~~~~~
  76 |     foo: Int;
```

Reply:
So, it is enough to check only the last field :)

Reply:
> If you write Slice? as remaining not in the last field, compiler will give an error already

We might re-enable using `as remaining` anywhere, it's actually can be quite useful

Reply:
Also, this makes this check orthogonal which is good from the design prospective

Reply:
Done